pkg/storage/storage_provisioner.go

```diff
@@ -42,14 +40,18 @@ type hostPathProvisioner struct {
 
 	// Identity of this hostPathProvisioner, generated. Used to identify "this"
 	// provisioner's PVs.
-	identity types.UID
+	identity string
 }
 
 // NewHostPathProvisioner creates a new Provisioner using host paths
 func NewHostPathProvisioner(pvDir string) controller.Provisioner {
+	nodeName := os.Getenv("NODE_NAME")
```

Comment:
who sets this environment variable ? and I dont see it being set anywhere ? 

also the name should be unique to minikube to not collide withe other tools, such as
MINIKUBE_STORAGE_PROVISIONER_NAME...or something like that